tests/conftest.py

```diff
@@ -27,6 +27,8 @@ def __init__(self):
         self.user_agent = "nexmo-python/{} python/{}".format(
             nexmo.__version__, platform.python_version()
         )
+        self.host = "rest.nexmo.com"
+        self.api_host = "api.nexmo.com"
```

Comment:
Since we have an error path that involves validating the host/api_host on see, we should have an off-nominal case in the tests to validate the setters throw the error correctly. Looks good other than that.